packages/playwright-core/src/server/webkit/wkPage.ts

```diff
@@ -627,27 +627,34 @@ export class WKPage implements PageDelegate {
     await this._page._onFileChooserOpened(handle);
   }
 
-  private static async _setEmulateMedia(session: WKSession, mediaType: types.MediaType | null, colorScheme: types.ColorScheme | null, reducedMotion: types.ReducedMotion | null, forcedColors: types.ForcedColors | null): Promise<void> {
+  private static async _setEmulateMedia(session: WKSession, mediaType: types.MediaType, colorScheme: types.ColorScheme, reducedMotion: types.ReducedMotion, forcedColors: types.ForcedColors): Promise<void> {
     const promises = [];
-    promises.push(session.send('Page.setEmulatedMedia', { media: mediaType || '' }));
+    promises.push(session.send('Page.setEmulatedMedia', { media: mediaType === 'no-override' ? '' : mediaType }));
     let appearance: any = undefined;
-    switch (colorScheme) {
-      case 'light': appearance = 'Light'; break;
-      case 'dark': appearance = 'Dark'; break;
+    if (colorScheme !== 'no-override') {
```

Comment:
If I had an override, and then call `page.emulateMedia({ colorScheme: null })`, it will be ignored here. Same for other browsers maybe?